### PR TITLE
Refactor to using TypeScript

### PR DESCRIPTION
**Pull request by TheDevMinerTV**

### CODE REVIEW

**Comment by TheDevMinerTV** [comment]
@loucass003 please review... this has been sitting here for too long :)

File: src/Tracker.ts [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+import { IncomingTapPacket } from './packets/IncomingTapPacket';
+import { IncomingTemperaturePacket } from './packets/IncomingTemperaturePacket';
+import { IncomingCorrectionDataPacket } from './packets/inspection/IncomingCorrectionDataPacket';
+import { IncomingFusedIMUDataPacket } from './packets/inspection/IncomingFusedIMUDataPacket';
+import { IncomingRawIMUDataPacket } from './packets/inspection/IncomingRawIMUDataPacket';
+import { OutgoingHandshakePacket } from './packets/OutgoingHandshakePacket';
+import { OutgoingPingPacket } from './packets/OutgoingPingPacket';
+import { OutgoingSensorInfoPacket } from './packets/OutgoingSensorInfoPacket';
+import { PacketWithSensorId } from './packets/Packet';
+import { parse } from './packets/PacketParser';
```
Comment (loucass003) [inline src/Tracker.ts:27]: Please make an index file in the packet folder. this is awfull

**Review by loucass003 — CHANGES_REQUESTED** [review]

File: src/Tracker.ts (hunk 1 above)
Comment (TheDevMinerTV) [reply src/Tracker.ts:27]: Soon, making the packets a library when this PR is merged

Comment (loucass003) [reply src/Tracker.ts:27]: Ok then

**Review by loucass003 — APPROVED** [review]

**Pull request merged**